src/generic/Matrix.jl

```diff
@@ -148,21 +148,7 @@ end
 #
 ###############################################################################
 
-# create a zero matrix
-function (a::Generic.MatSpace{T})() where {T <: NCRingElement}
```

Comment:
These days `Generic.MatSpace` is the only subtype of `MatSpace` so perhaps we should just merge the two

Reply:
See https://github.com/Nemocas/AbstractAlgebra.jl/issues/1840